test/results/no-default-headers-with-no-curl-flag-expected.txt

```diff
@@ -0,0 +1,242 @@
+      name: "bytes"
+      number: 7
+      label: LABEL_OPTIONAL
+      type: TYPE_BYTES
+      json_name: "bytes"
+    }
+    field: {
+      name: "fooEnum"
+      number: 8
+      label: LABEL_OPTIONAL
+      type: TYPE_ENUM
+      type_name: ".happyday.Foo"
+      json_name: "fooEnum"
+    }
+    field: {
+      name: "misc"
+      number: 9
+      label: LABEL_REPEATED
+      type: TYPE_MESSAGE
+      type_name: ".happyday.MiscInfo"
+      json_name: "misc"
+    }
+    field: {
+      name: "float"
+      number: 10
+      label: LABEL_OPTIONAL
+      type: TYPE_FLOAT
+      json_name: "float"
+    }
+    field: {
+      name: "NonCamel_case_FieldName"
+      number: 11
+      label: LABEL_OPTIONAL
+      type: TYPE_STRING
+      json_name: "NonCamelCaseFieldName"
+    }
+  }
+  message_type: {
+    name: "HappyDayResponse"
+    field: {
+      name: "isHappyDay"
+      number: 1
+      label: LABEL_OPTIONAL
+      type: TYPE_BOOL
+      json_name: "isHappyDay"
+    }
+    field: {
+      name: "reason"
+      number: 2
+      label: LABEL_OPTIONAL
+      type: TYPE_STRING
+      json_name: "reason"
+    }
+    field: {
+      name: "formattedDate"
+      number: 3
+      label: LABEL_OPTIONAL
+      type: TYPE_STRING
+      json_name: "formattedDate"
+    }
+    field: {
+      name: "err"
+      number: 4
+      label: LABEL_OPTIONAL
+      type: TYPE_STRING
+      json_name: "err"
+    }
+  }
+  message_type: {
+    name: "MiscInfo"
+    field: {
+      name: "weatherOfPastFewDays"
+      number: 1
+      label: LABEL_REPEATED
+      type: TYPE_STRING
+      json_name: "weatherOfPastFewDays"
+    }
+    field: {
+      name: "fooString"
+      number: 2
+      label: LABEL_OPTIONAL
+      type: TYPE_STRING
+      oneof_index: 0
+      json_name: "fooString"
+    }
+    field: {
+      name: "fooEnum"
+      number: 3
+      label: LABEL_OPTIONAL
+      type: TYPE_ENUM
+      type_name: ".happyday.Foo"
+      oneof_index: 0
+      json_name: "fooEnum"
+    }
+    oneof_decl: {
+      name: "alternative"
+    }
+  }
+  enum_type: {
+    name: "Foo"
+    value: {
+      name: "BAR"
+      number: 0
+    }
+    value: {
+      name: "BAZ"
+      number: 1
+    }
+    value: {
+      name: "FAZ"
+      number: 2
+    }
+  }
+  syntax: "proto3"
+}
+Looking up message with full name: happyday.HappyDayRequest
+Looking up message with full name: happyday.HappyDayRequest
+=========================== Request Text     =========================== >>>
+date: {
+  seconds: 1648044939
+  nanos: 152000000
+}
+includeReason: true
+=========================== Request Binary   =========================== >>>
+00000000  0a 0b 08 8b d7 ec 91 06  10 80 ac bd 48 10 01     |............H..|
+Using internal http request due to forced avoidance of curl.
+Invoking internal http request.
+######### STDERR #########
+Error: Custom headers are not supported when  using internal http. Please provide curl in path and avoid using --no-curl. Found headers: []
```

Comment:
We need to adapt the error message in this case, so that it also includes the new option.

e.g. "Non-default or custom headers..."